services/question-service/src/entry-points/api/routes.ts

```diff
@@ -22,6 +22,15 @@ export default function defineRoutes(expressApp: express.Application) {
     }
   });
 
+  router.post("/bulk", async (req, res, next) => {
+    try {
+      const addQuestionResponse = await questionUseCase.addQuestions(req.body);
+      res.json(addQuestionResponse);
+    } catch (error) {
+      next(error);
+    }
+  });
+
```

Comment:
What happens if there is a duplicate question in the incoming questions? i.e. same `title` or same `link`?

Reply:
It will be caught by the schema and none of the post request will go through

Reply:
It's either this or we can loop through the array and add it one by one.